demo/App.js

```diff
@@ -1,12 +1,177 @@
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { AppLoading, Asset, Font } from "expo";
+import "ethers/dist/shims.js";
+import { ethers } from "ethers";
+
 
 export default class App extends React.Component {
   state = {
     isLoadingComplete: false
   };
 
+
+	componentDidMount() {
+
+		let ropstenProvider = ethers.getDefaultProvider('ropsten');
+		let mainnetProvider = ethers.getDefaultProvider();
+		let utils = ethers.utils;
+
+		let privateKey = "0x3141592653589793238462643383279502884197169399375105820974944592";
+		let address = "0x7357589f8e367c2C31F51242fB77B350A11830F3";
+		let wallet = new ethers.Wallet(privateKey);
+		let fundedWallet = new ethers.Wallet("0x18bfdd05c1e63e4a27081352e2910c164a4d34588f8d7eecfdfcb654742934c2", ropstenProvider); //addr:0xa9adeF68CEF4EA3C9d0717126e14fDfebFDFB4d7
+
+		let wallet2mnemonic = "radar blur cabbage chef fix engine embark joy scheme fiction master release";
+		let wallet2 = ethers.Wallet.fromMnemonic(wallet2mnemonic);
+		let wallet3 = ethers.Wallet.createRandom();
+
+		console.assert(wallet.address === address, "wallet.address");
+		console.assert(wallet.privateKey === privateKey, "wallet.privateKey");
+		console.assert(wallet2.mnemonic === wallet2mnemonic, "wallet.privateKey");
+		console.assert(wallet3.address !== undefined, "wallet.address from randonly created");
```

Comment:
@marcelomorgado Do you mean `rand only` or `randomly`?

Reply:
randomly. Sorry